src/legacy/Routes.tsx

```diff
@@ -57,6 +57,9 @@ class RouteApp extends React.Component<any, any> {
       errorModalData: new ErrorModalData(),
```

Comment:
All the state changes in this file are the legacy antipattern we want to avoid. Let's have all this moved into a redux reducer ("slice" in terms of @reduxjs/toolkit), and here, if necessary, call `dispatch` functions to populate QRCodeGenerator state.

Reply:
@br4in3x i have updated follow your mention